src/components/PrivateRoute.jsx

```diff
@@ -1,19 +1,55 @@
-import React from 'react';
+import '@reshuffle/code-transform/macro';
+import React, { useState, useEffect } from 'react';
 import { Route, Redirect } from 'react-router-dom';
+import { useAuth } from '@reshuffle/react-auth';
+import { validateUser } from '../../backend/authBackend';
 
 /**
- * Standard React-router route that requires a userToken
+ * Standard React-router route that requires reshuffle identity
  */
-const PrivateRoute = ({ props, component: Component, userToken, ...rest }) => {
+const PrivateRoute = (props) => {
+  const { component: Component, ...rest } = props;
+  const { authenticated } = useAuth();
+  const [email, setEmail] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const value = await validateUser();
+        setEmail(value);
+
+      } catch (err) {
+        console.error(err);
+        setEmail(false)
```

Comment:
Why do you set email here?

Reply:
I see now that it is redundant, removing since it is already false